File: GameArchitecture/EntityComponentSystems_Inspiration/XNA Component Entity System OLD/xnaentitycomponents_48724efaffb5/XNAEntityComponents/XNAEntityComponents/Components/TransformComponent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Xna.Framework;
using XNAEntityComponents.EntitySystem;

namespace XNAEntityComponents.Components
{
    public class TransformComponent : EntityComponent
    {
        public Vector2 Position;
        public float Scale;
        public float Rotation;

        public TransformComponent(Entity aParent) : base(aParent) 
        {
            this.Name = "TransformComponent";
        }
    }
}

File: GameArchitecture/Project1/core/src/net/gustavdahl/projectone/EntityComponentSystem/TransFormComponent.java
package net.gustavdahl.projectone.EntityComponentSystem;

import com.badlogic.gdx.math.Vector2;

public class TransFormComponent extends Component
{

	Vector2 Position;
	float Rotation;
	Vector2 Scale;

	public TransFormComponent()
	{
		Position = new Vector2(0,0);
		Rotation = 0f;
		Scale = new Vector2(1,1);
	}
	
	@Override
	public String Name()
	{
		return "Transform";
	}

}
